x-pack/plugins/uptime/public/components/monitor/ml/ml_job_link.test.tsx

```diff
@@ -5,28 +5,17 @@
  */
 
 import React from 'react';
-import { coreMock } from 'src/core/public/mocks';
-import { renderWithRouter, shallowWithRouter } from '../../../lib';
+import { renderTLWithRouter } from '../../../lib';
 import { MLJobLink } from './ml_job_link';
-import { KibanaContextProvider } from '../../../../../../../src/plugins/kibana_react/public';
 
-const core = coreMock.createStart();
 describe('ML JobLink', () => {
-  it('shallow renders without errors', () => {
-    const wrapper = shallowWithRouter(
-      <MLJobLink dateRange={{ to: '', from: '' }} basePath="" monitorId="testMonitor" />
-    );
-    expect(wrapper).toMatchSnapshot();
-  });
-
   it('renders without errors', () => {
-    const wrapper = renderWithRouter(
-      <KibanaContextProvider
-        services={{ ...core, triggersActionsUi: { getEditAlertFlyout: jest.fn() } }}
-      >
-        <MLJobLink dateRange={{ to: '', from: '' }} basePath="" monitorId="testMonitor" />
-      </KibanaContextProvider>
+    const { asFragment } = renderTLWithRouter(
+      <MLJobLink dateRange={{ to: '', from: '' }} basePath="" monitorId="testMonitor" />,
```

Comment:
This is an example where we needed to extend the core options. You can do so by passing `customCoreOptions`. If you need to overwrite the entire core and make your own mock, you can do so by passing `{ kibanaProps: { services: { ...myCustomCore } } }`.

Reply:
I'm unable to figure out why this component needed to call `getEditAlertFlyout`. The `MLJobLink` component sets up an href and a few other fields and renders a link.